client/src/Signup/Confirmation.js

```diff
@@ -1,9 +1,57 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { Divider, Typography } from 'antd'
+import LabelImportantIcon from '@material-ui/icons/LabelImportant'
 
-export default function Confirmation() {
+const { Title, Text, Link } = Typography
+
+const userEmail = 'chelsea@pieforproviders.com'
+const pieEmail = 'noreply@pieforproviders.com'
+
+function ListItem({ children }) {
   return (
-    <div className="confirmation">
-      <h1>Check your email for confirmation</h1>
+    <div className="flex align-center justify-left mb-1">
+      <LabelImportantIcon
+        className="mr-1"
+        style={{ color: '#000', width: '16px' }}
+      />
+      {children}
     </div>
   )
 }
+
+export default function Confirmation() {
```

Comment:
looking at the code we've been following component arrow functions as opposed to functions!